colossalai/fx/profiler/dataflow.py

```diff
@@ -0,0 +1,92 @@
+from typing import Dict
+from torch.fx import Graph, Node
+from .memory import INPLACE_ATEN, activation_size
+
+
+def is_forward(n: Node):
+    assert 'stage' in n.meta, f'Node meta of {n} has no key `stage`!'
+    return n.meta['stage'] == 'f'
```

Comment:
Do not use a string to represent the stage. You should construct a enum instead.